<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cfenetreajouttab</class>
 <widget class="QDialog" name="cfenetreajouttab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="plFATlabelnomtab">
      <property name="text">
       <string>Nom de la tabulation : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pleFATlineeditnomdelatabulation"/>
    </item>
    <item>
     <widget class="QLabel" name="plFATlabelnombrecolone">
      <property name="text">
       <string>Nombre de colonnes : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pleFATlineeditnombredecolonne"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>241</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="ppbFATboutonok">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ppbFATboutonannuler">
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
